aries_cloudagent/ledger/multiple_ledger/indy_vdr_manager.py

```diff
@@ -94,11 +94,12 @@ async def set_profile_write_ledger(self, ledger_id: str, profile: Profile) -> st
                 profile.context.settings["endorser.endorser_public_did"] = endorser_did
             profile.context.settings["ledger.write_ledger"] = ledger_id
             if multi_tenant_mgr:
-                extra_settings = {
-                    "endorser.endorser_alias": endorser_alias,
-                    "endorser.endorser_public_did": endorser_did,
-                    "ledger.write_ledger": ledger_id,
-                }
+                # may not use an endorser, so check first...
+                if endorser_info:
```

Comment:
May be this can be moved under [existing endorser_info check](https://github.com/hyperledger/aries-cloudagent-python/blob/7b1752a98bddc226cb6e4a75f821147dff7563c7/aries_cloudagent/ledger/multiple_ledger/indy_vdr_manager.py#L94)